internal/controllers/machineset/machineset_delete_policy.go

```diff
@@ -34,10 +34,11 @@ type (
 )
 
 const (
-	mustDelete    deletePriority = 100.0
-	betterDelete  deletePriority = 50.0
-	couldDelete   deletePriority = 20.0
-	mustNotDelete deletePriority = 0.0
+	mustDelete         deletePriority = 100.0
+	almostAlwaysDelete deletePriority = 75.0
```

Comment:
let me know if you have a better idea for naming this :)

Reply:
what about 
```suggestion
	shouldDelete deletePriority = 75.0
```

Reply:
yeah that sounds better, I changed it